sdk/playwrighttesting/microsoft-playwright-testing/src/utils/serviceClient.ts

```diff
@@ -61,8 +61,10 @@ export class ServiceClient {
     if (response.status === 200) {
       return JSON.parse(response.bodyAsText!) as TestRun;
     } else {
```

Comment:
if you are throwing the error outside, then else block is not required. Everything can be moved post if block.